ibis/expr/window.py

```diff
@@ -228,10 +274,31 @@ def trailing_window(periods, group_by=None, order_by=None):
     -------
     win : ibis Window
     """
-    return Window(preceding=periods, following=0,
+    return Window(preceding=rows, following=0,
                   group_by=group_by, order_by=order_by)
 
 
+def trailing_time_window(preceding, order_by, group_by=None):
+    """
+    Create a trailing time window for use with aggregate window functions.
+
+    Parameters
+    ----------
+    preceding : float or expression of intervals, i.e.
+      1 * ibis.day() + 5 * ibis.hour()
+    order_by : Union[ir.TimeColumn, ir.DateColumn, ir.TimestampColumn]
+      Specify the column over which the window will look back
+    group_by : expressions, default None
+      Either specify here or with TableExpr.group_by
+
+    Returns
+    -------
+    win: ibis Window
+    """
+    return Window(preceding=preceding, following=0,
+                  group_by=group_by, order_by=order_by, how='time_range')
```

Comment:
Can we do this without a special value of `how`? Can this be deduced by the fact that `order_by` is a timestamp or date column and any of the `preceding` and `following` arguments must be interval expressions or `pandas.Timedelta` objects?